src/Symfony/Component/Security/Http/Authenticator/FormLoginAuthenticator.php

```diff
@@ -157,6 +157,10 @@ private function getCredentials(Request $request): array
 
         $request->getSession()->set(Security::LAST_USERNAME, $credentials['username']);
 
+        if (!\is_string($credentials['password'])) {
```

Comment:
for the username we allow "stringable" as well. Is this relevant for the password? I guess not, or?

Reply:
But is that a real use-case? I mean do people have value objects for their passwords in the request bag somehow? :thinking:

Reply:
I've seen codebases where value objects were used to "protect" sensitive data (e.g. they throw if you try to serialize them). I don't know if anyone uses this kind of objects in this context. But if someone does we'd break their app with an LTS bugfix update which would be unfortunate. Let's have the authenticator accept anything that can be cast to a string.